example_dags/calculate_popular_movies.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
```

Comment:
Isn't this DAG just a subset of https://github.com/astronomer/astro-sdk/blob/main/example_dags/example_transform.py ?

We could just use that

Reply:
@sunank200 There was some other code added to the example_transform.py and I wasn't sure if we can use that in README.md. I think we can take @mikeshwe opinion on this.